add Github issue concourse resource (#6)
openstax/concourse-resources: github-issue/README.md

# Github Issue Resource

Fetches Github Issue

## Source Configuration

* `token`: *Required.* github token.

* `repository`: *Required.* The repository name [organization/name].

> * `version: {"number": <issue number> , "modified": <last modified date of issue>}`: 

Reviewer: this is not in the source anymore right?

### Example

``` yaml
- name: github-issue
  type: github-issue
  source:
    repository: 'org/repo'
    token: 'token'
  version: {"number": <number> , "modified": <date>}
```

``` yaml
- get: github-issue
```

## Behavior

### `check`: Check for new issues.

> Checks to see if an issue has been updated recently, and if so, passes to `in`. You can specify a specific issue if needed as 

Reviewer: ```suggestion
Checks to see if any issues has been updated or created, and if so, passes to `in`.
```

> A version can be passed in if needed. 

Reviewer: ```suggestion
```
